cg/meta/workflow/taxprofiler.py

```diff
@@ -104,3 +104,9 @@ def get_multiqc_search_patterns(self, case_id: str) -> dict:
     def get_genome_build(self, case_id: str) -> str:
         """Return the reference genome build version of a Taxprofiler analysis."""
         return GenomeVersion.T2T_CHM13.value
+
+    @property
+    def use_read_count_threshold(self) -> bool:
```

Comment:
I'm not sure this method is used somewhere, seems like it's just defined in the nextflow analysis api but it's never called 🤔

Reply:
the parent method was removed there https://github.com/Clinical-Genomics/cg/pull/2886/files
so it has to be adapted on the nf-analysis class or has already be done and can just be removed

Reply:
I can remove those methods and test `start-available` for all pipelines (rare disease, tomte, taxprofiler, rnafusion)

Reply:
not the test. Only the methods. I'm not sure if the behaviour is correct right now. Can you check?

Reply:
https://github.com/Clinical-Genomics/cg/blob/b6ad74c3a62d6b829475ca37f85bf39fcfe1d0c9/cg/services/quality_controller/quality_checks/checks.py#L25-L53

It is correct.